lib/session/sessionManager.ts

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+import { OnSessionMessage, SessionHandlerOptions, MessageSession } from "./messageSession";
+import { OnError } from "../core/messageReceiver";
+import { ClientEntityContext } from "../clientEntityContext";
+import { getProcessorCount } from "../util/utils";
+import * as log from "../log";
+
+export class SessionManager {
+  /**
```

Comment:
Have added very basic stuff. This needs more changes to ensure things work as expected.